app/src/main/java/com/android/partagix/ui/screens/ViewAccount.kt

```diff
@@ -0,0 +1,183 @@
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material.icons.filled.CheckCircle
+import androidx.compose.material.icons.filled.LocationOn
+import androidx.compose.material3.Button
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.android.partagix.R
+import com.android.partagix.model.UserViewModel
+import com.android.partagix.ui.components.BottomNavigationBar
+import com.android.partagix.ui.navigation.NavigationActions
+import com.android.partagix.ui.navigation.Route
+import kotlin.math.round
+
+// @Preview(showBackground = true)
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ViewAccount(
+    modifier: Modifier = Modifier,
+    navigationActions: NavigationActions,
+    userViewModel: UserViewModel,
+) {
+  val uiState = userViewModel.uiState.collectAsState()
+  val user = uiState.value.user
+  Scaffold(
+      modifier = Modifier.fillMaxSize().testTag("viewAccount"),
+      topBar = {
+        TopAppBar(
+            title = { Text("My Account") },
+            modifier = Modifier.fillMaxWidth().testTag("title"),
+            navigationIcon = {
+              IconButton(onClick = { navigationActions.goBack() }) {
+                Icon(
+                    imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+                    contentDescription = null,
+                    modifier = Modifier.width(48.dp))
+              }
+            })
+      },
+      bottomBar = {
+        BottomNavigationBar(
+            selectedDestination = Route.ACCOUNT,
+            navigateToTopLevelDestination = navigationActions::navigateTo,
+            modifier = modifier.testTag("accountScreenBottomNavBar"))
+      }) {
+        Column(
+            modifier =
+                Modifier.fillMaxHeight()
+                    .padding(it)
+                    .verticalScroll(rememberScrollState())
+                    .testTag("mainContent")) {
+              /*LaunchedEffect(key1 = uiState) {
+                  println("refreshed User: $user")
+                  user = uiState.user
+
+              }*/
+              Image(
+                  painter =
+                      painterResource(
+                          id = R.drawable.ic_launcher_background) /*TODO: get profile picture*/,
```

Comment:
"TODO: get profile picture" not part of this task ?

Reply:
No, the database does not allow to store images yet